src/frontend/pages/recruitment/Configuration/ConfigurationPage.js

```diff
@@ -208,8 +294,8 @@ const ConfigurationPage = () => {
                   <LocalizationProvider dateAdapter={AdapterDateFns}>
                     <DesktopDatePicker
                       inputFormat="MM/dd/yyyy"
```

Comment:
Don't know how this made it through the previous PR, but can you change the format so that the day isn't between the month and the year?

Reply:
Sure, I'll format it `dd/MM/yyyy`. Let me know if this works with you.